packages/fractor-typoscript/src/Contract/TypoScriptNodeVisitor.php

```diff
@@ -13,7 +13,7 @@
 interface TypoScriptNodeVisitor
 {
     /**
-     * @param list<Statement> $statements
```

Comment:
`list<Statement>` has different (stronger) semantics than `Statement[]` – the former is an indexed list, the latter is just an array w/o revealing anything about the keys. Are we sure we want this change here? Or is it PhpStan complaining for some reason?

Reply:
The problem is that the TypoScript Parser (third party package) has the annotation `@return Statement[] The syntax tree.` for the method "parseString".